lib/onfido/resource.rb

```diff
@@ -51,7 +52,11 @@ def make_request(options)
     end
 
     def parse(response)
-      JSON.parse(response.body.to_s)
+      if response.headers[:content_type] == "application/octet-stream"
```

Comment:
It feels like it'd be better to reverse this and parse JSON if its verifiably JSON, and otherwise return the raw body. This would mean that if another `Content-Type` came back (say, an `application/pdf` for a hypothetical API that lets you fetch a downloadable copy of a report), it would blow up. It's probably safer to return the raw response *unless* it's JSON.

```ruby
if response.headers[:content_type] == "application/json"
  JSON.parse(response.body.to_s)
else
  response.body
end
```